packages/react-examples/src/react/Icon/docs/IconOverview.md

```diff
@@ -2,4 +2,4 @@ In a user interface, an icon is an image that represents an application, a capab
 
 For a list of icons, visit our [icon documentation](#/styles/web/icons).
 
-Note that icons are not bundled by default and typically must be loaded by calling `initializeIcons` from the `@uifabric/icons` package at the root of your application. See the [icon documentation](#/styles/web/icons#fabric-react) for more details.
+Note that icons are not bundled by default and typically must be loaded by calling `initializeIcons` from the `@fluentui/font-icons-mdl2` package at the root of your application. See the [icon documentation](#/styles/web/icons#fabric-react) for more details.
```

Comment:
Not a new issue but we should probably discourage people from referencing the package directly
```suggestion
Note that icons are not bundled by default and typically must be loaded by calling `initializeIcons` from `@fluentui/react/lib/Icons` at the root of your application. See the [icon documentation](#/styles/web/icons#fabric-react) for more details.
```